mocks/driver/sql/notification_queue.go

```diff
@@ -0,0 +1,127 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Could you add the command to generate this into [mocks.go](https://github.com/hellofresh/goengine/blob/master/mocks/mocks.go#L25)?